From 9470d00850e8b622d963d4d9c61ad1e59972310a Mon Sep 17 00:00:00 2001
From: Madhurkiran Harikrishnan <madhurkiran.harikrishnan@xilinx.com>
Date: Tue, 28 Jan 2020 16:10:54 -0800
Subject: [PATCH 1/2] src: egl: eglinfo: Align EXT_platform_device extension
 macros with khronos

Although the macro definitions are consistent, using the macros name
defined under khronos makes it backward compatible with older eglext.h

Signed-off-by: Madhurkiran Harikrishnan <madhurkiran.harikrishnan@xilinx.com>
Upstream-Status: Inappropriate [Xilinx specific]
Signed-off-by: Mark Hatle <mark.hatle@amd.com>
---
 src/egl/opengl/eglinfo.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/egl/opengl/eglinfo.c b/src/egl/opengl/eglinfo.c
index 72fe45a..e8a0225 100644
--- a/src/egl/opengl/eglinfo.c
+++ b/src/egl/opengl/eglinfo.c
@@ -305,17 +305,17 @@ main(int argc, char *argv[])
                                                   NULL), "Android platform");
        if (strstr(clientext, "EGL_MESA_platform_gbm") ||
            strstr(clientext, "EGL_KHR_platform_gbm"))
-           ret += doOneDisplay(getPlatformDisplay(EGL_PLATFORM_GBM_MESA,
+           ret += doOneDisplay(getPlatformDisplay(EGL_PLATFORM_GBM_KHR,
                                                   EGL_DEFAULT_DISPLAY,
                                                   NULL), "GBM platform");
        if (strstr(clientext, "EGL_EXT_platform_wayland") ||
            strstr(clientext, "EGL_KHR_platform_wayland"))
-           ret += doOneDisplay(getPlatformDisplay(EGL_PLATFORM_WAYLAND_EXT,
+           ret += doOneDisplay(getPlatformDisplay(EGL_PLATFORM_WAYLAND_KHR,
                                                   EGL_DEFAULT_DISPLAY,
                                                   NULL), "Wayland platform");
        if (strstr(clientext, "EGL_EXT_platform_x11") ||
            strstr(clientext, "EGL_KHR_platform_x11"))
-           ret += doOneDisplay(getPlatformDisplay(EGL_PLATFORM_X11_EXT,
+           ret += doOneDisplay(getPlatformDisplay(EGL_PLATFORM_X11_KHR,
                                                   EGL_DEFAULT_DISPLAY,
                                                   NULL), "X11 platform");
        if (strstr(clientext, "EGL_MESA_platform_surfaceless"))
-- 
2.17.1

